check-submodule.py

```diff
@@ -247,7 +305,7 @@ def compute_metrics(args: argparse.Namespace) -> None:
     old_dir.mkdir(parents=True, exist_ok=True)
     new_dir.mkdir(parents=True, exist_ok=True)
 
-    # Skip if only new metrics are requested
+    # Skip whether only new metrics are requested
```

Comment:
The previous sentence was correct (even though it doesn't sound great), this one is not.
Maybe use `when` instead of `if`.

Note: this change should be in a separate commit/PR, as it is unrelated to this commit

Reply:
That was a reminiscence of an old change, sorry :P